### DISCUSSION TITLE
HW 2 Part 3.2

### DISCUSSION THREAD

**Discussion by Aarons-Joshua** [cooperrc/computational-mechanics#44] [Q&A]
Has anyone got to this problem yet? If so, can anyone help when it comes to the linear regression section. I keep getting this really weird error, and I am unsure how to resolve it. Here is my code below:

![image](https://user-images.githubusercontent.com/98834673/172754943-1f60ebd8-77aa-4baa-94a7-46999b811b5f.png)
![image](https://user-images.githubusercontent.com/98834673/172754998-c927f1a5-e46e-493e-b21c-8571f92e1c2a.png)

**Comment by Aidan-Myatt** [comment]
This is what worked for me, but I am not fully confident on how correct it is. The difference is the way the function is defined. I tried to define it like you did; however, it gave me an error. Therefore, I defined in linearly like it says in the problem statement. 

![image](https://user-images.githubusercontent.com/106316906/172857513-b17f2da4-ff93-4b64-9b67-7f289b777e8e.png)
